template/content/server.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+'use strict';
```

Comment:
This module doesn't do much and in fact, I think it leads to some initial confusion about what the distinction between app.js and server.js are.